src/src/auth/auth.service.ts

```diff
@@ -128,4 +132,55 @@ export class AuthService {
 
 
     }
+
+    public async resetPassword(req, response) {
```

Comment:
- we should also get from request mobile, OTP, reason, hash, pwd, confirm pwd all
- validate OTP again
- then change pwd

Reply:
done